tests/zfs-tests/include/default.cfg.in

```diff
@@ -209,6 +209,8 @@ if is_linux; then
 	ZVOL_RDEVDIR="/dev/zvol"
 	DEV_RDSKDIR="/dev"
 	DEV_MPATHDIR="/dev/mapper"
+	ZEDLET_DIR="/var/tmp/zed"
+	ZFS_INTREE_DIR="/var/tmp/zfs"
```

Comment:
I'd prefer not to add this additional variable.  Let's just store the temporary `vdev_id.conf` in `$ZEDLET_DIR`.